plugin/trino-hive/src/main/java/io/trino/plugin/hive/parquet/ParquetPageSource.java

```diff
@@ -230,4 +233,11 @@ private static Block getRowIndexColumn(long baseIndex, int size)
         }
         return new LongArrayBlock(size, Optional.empty(), rowIndices);
     }
+
+    @Override
+    public Metrics getMetrics()
+    {
+        return new Metrics(parquetReader.getCodecsUsed().stream()
+                .collect(toImmutableMap(codec -> codec, codec -> new LongCount(0))));
```

Comment:
We have metics names, but we don't have metrics values.
What is the value supposed to mean?

Reply:
I've updated the value to be the data length as per this comment: https://github.com/trinodb/trino/pull/12551#discussion_r882354106